lib/ansible/playbook/play_context.py

```diff
@@ -616,3 +618,29 @@ def update_vars(self, variables):
                         variables[var_opt] = var_val
             except AttributeError:
                 continue
+
+    @property
+    def connection(self):
```

Comment:
Should `connection` property not be named differently than the `connection` attribute ? It seems the property will be overridden by the `_connection` `FieldAttribute` defined in `Base` and never used/called.